components/form/form-element-mixin.js

```diff
@@ -92,12 +87,15 @@ export const FormElementMixin = superclass => class extends LocalizeCoreElement(
 			name: { type: String },
 			noValidate: { type: Boolean, attribute: 'novalidate' },
 			validationError: { type: String, attribute: false },
+			childErrors: { type: Object, attribute: false },
+			_errors: { type: Array, attribute: false }
```

Comment:
Just wanted to confirm that a change to either of these needs to force a re-render of the component into which `FormElementMixin` is mixed in? `_errors` seems like a definite yes, but doesn't `childErrors` typically just impact parents, which would be communicated via the `d2l-form-element-errors-change` event? If so, it could just be a private variable vs. a property.

Reply:
`_errors` is actually the one we could get away with without it being a property since `validationError` is a property that references the first error in `_errors` so a change to `_errors` is already guaranteed to cause a rerender.

`childErrors` maintains a mapping of errors that exist in the component's nested from elements. If `childErrors` changes we need to rerender because the parent may want to hide or show its tooltip in response to the child hiding or showing its tooltip.